excise/utils.py

```diff
@@ -386,12 +385,14 @@ def get_checkout_lines_data(
             product_type=line_info.product_type
         )
 
-    append_shipping_to_data(
-        data,
-        config.shipping_product_code,
-        shipping_address,
-        checkout_info.shipping_method_channel_listings,
-    )
+    if checkout_info.delivery_method_info.delivery_method:
+        price = checkout_info.delivery_method_info.delivery_method.price
+        append_shipping_to_data(
+            data,
+            config.shipping_product_code,
+            shipping_address,
+            price.amount if price else None,
```

Comment:
What happens when this is `None`? Have you tested this case?

Reply:
There is a condition in `append_shipping_to_data` too: https://github.com/saleor/saleor-plugin-avatax-excise/pull/15/files#diff-78427328fef6491b85480ce65bad3a477c873d86f526563b229f1f8795cf91c1R302.